zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedKubernetesClusterChecker.scala

```diff
@@ -16,29 +18,57 @@ object DeletedKubernetesClusterChecker {
   def impl[F[_]](
     dbReader: DbReader[F],
     deps: KubernetesClusterCheckerDeps[F]
-  )(implicit F: Concurrent[F], logger: Logger[F], ev: Ask[F, TraceId]): CheckRunner[F, K8sClusterToScan] =
-    new CheckRunner[F, K8sClusterToScan] {
+  )(implicit F: Concurrent[F], logger: Logger[F], ev: Ask[F, TraceId]): CheckRunner[F, KubernetesCluster] =
+    new CheckRunner[F, KubernetesCluster] {
       override def appName: String = zombieMonitor.appName
 
-      override def resourceToScan: Stream[F, K8sClusterToScan] = dbReader.getk8sClustersToDeleteCandidate
+      override def resourceToScan: Stream[F, KubernetesCluster] = dbReader.getk8sClustersToDeleteCandidate
 
       override def configs = CheckRunnerConfigs(s"deleted-kubernetes", false)
 
       override def dependencies: CheckRunnerDeps[F] = deps.checkRunnerDeps
 
-      def checkResource(cluster: K8sClusterToScan, isDryRun: Boolean)(implicit
+      def checkResource(cluster: KubernetesCluster, isDryRun: Boolean)(implicit
         ev: Ask[F, TraceId]
-      ): F[Option[K8sClusterToScan]] =
+      ): F[Option[KubernetesCluster]] = {
+        val isZombie = cluster.cloudContext match {
```

Comment:
Actual functional change in this file: it's now checking both GCP and Azure k8s clusters for zombie-ness.